homeworks/src/ltretykhlib/module3/Currency.java

```diff
@@ -0,0 +1,30 @@
+package ltretykhlib.module3;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+/**
+ * Created by Tretykhlib on 23.04.2017.
+ */
+public class Currency {
+
+    private double rate;
+
+    public Currency(double convertRate) {
+        rate = convertRate;
+    }
+
+
+    public BigDecimal convert(int uahAmount) {
+        double usdAmount = uahAmount / rate;
```

Comment:
use BigDecimal#divide method 

   private final BigDecimal rate;
   public Currency(double convertRate) {
        this.rate = new BigDecimal(convertRate);
    }

// or just make the method to take a parameter of BigDecimal type
    public BigDecimal convert(double uahAmount) {
        return new BigDecimal(uahAmount).divide(rate, 2, RoundingMode.HALF_EVEN);
    }